jkube-kit/config/service/src/main/java/org/eclipse/jkube/kit/config/service/ApplyService.java

```diff
@@ -939,6 +926,17 @@ public void applyService(Service service, String sourceName) throws Exception {
         }
     }
 
+    private <T extends HasMetadata> void doPatchEntity(T oldEntity, T newEntity, String namespace, String sourceName) {
+        String kind = newEntity.getKind();
+        log.info("Updating {} from {}", kind, sourceName);
```

Comment:
I'm not sure `KitLogger` supports format specifiers

Reply:
It does, but using String Formatter placeholders, use following `log.info("Updating %s from %s", kind, sourceName);

Reply:
Thanks for your suggestions, I implemented them all.